model/SimpleLicensing/Properties/licenseListVersion.md

```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# licenseListVersion
+
+## Summary
+
+The version of the SPDX License List used when the license expression was created.
+
+## Description
+
+Recognizing that licenses are added to the SPDX License List with each subsequent version, the intent is to provide consumers with the version of the SPDX License List used. This anticipates that in the future, license expression might have used a version of the SPDX License List that is older than the then current one.
```

Comment:
"The specific version of the SPDX License List must include all listed licenses and exceptions used in the expression." -- or something like that.

Reply:
Updated